### PR TITLE
Toolkit Migration Nodejs PR

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/bot-sharepoint-file-viewer/nodejs/build/aad.manifest.local.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
+    "id": "43b86700-751c-4846-91b3-00081b402d0d",
```
Comment (Prithvi-MSFT) [inline samples/bot-sharepoint-file-viewer/nodejs/build/aad.manifest.local.json:2]: remove from all samples

File: samples/bot-archive-groupchat-messages/nodejs/build/aad.manifest.dev.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
+    "id": "c2907487-6fcf-4c11-8ab0-34d22cabc46f",
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/bot-archive-groupchat-messages/nodejs/build/aad.manifest.dev.json:2]: fixed

File: samples/bot-sharepoint-file-viewer/nodejs/build/aad.manifest.dev.json [hunk 3]
```diff
@@ -0,0 +1,107 @@
+{
+    "id": "43b86700-751c-4846-91b3-00081b402d0d",
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/bot-sharepoint-file-viewer/nodejs/build/aad.manifest.dev.json:2]: fixed

File: samples/bot-sharepoint-file-viewer/nodejs/build/aad.manifest.local.json (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/bot-sharepoint-file-viewer/nodejs/build/aad.manifest.local.json:2]: fixed

File: samples/graph-activity-feed/nodejs/build/aad.manifest.local.json [hunk 4]
```diff
@@ -0,0 +1,109 @@
+{
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/graph-activity-feed/nodejs/build/aad.manifest.local.json:1]: fixed

File: samples/graph-activity-feed/nodejs/build/aad.manifest.dev.json [hunk 5]
```diff
@@ -0,0 +1,137 @@
+{
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/graph-activity-feed/nodejs/build/aad.manifest.dev.json:1]: fixed

File: samples/graph-activity-feed-broadcast/nodejs/build/aad.manifest.local.json [hunk 6]
```diff
@@ -0,0 +1,137 @@
+{
+    "id": "20d8d630-a271-447d-b28e-c9a9ad618a50",
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/graph-activity-feed-broadcast/nodejs/build/aad.manifest.local.json:2]: 1fixed

File: samples/graph-activity-feed-broadcast/nodejs/infra/azure.bicep
Comment (formulahendry) [inline samples/graph-activity-feed-broadcast/nodejs/infra/azure.bicep:1]: this is a tab project? Then Azure Bot is not needed.

Comment (Mohammed-MSFT) [reply samples/graph-activity-feed-broadcast/nodejs/infra/azure.bicep:1]: Yeah, its not a bot project but first we tried with tab formula its creating domain like localhost:someport..
with that configure tab unable to load and didn't get the page.

the second time tried with bot formula it has generated domain like devtunel and that's works for this sample
let us know if you want us to revert changes for this sample.
Thanks!!
 
notify-feed-2

Comment (formulahendry) [reply samples/graph-activity-feed-broadcast/nodejs/infra/azure.bicep:1]: If it is not a bot project, please remove unrelated Bot things.

Comment (Mohammed-MSFT) [reply samples/graph-activity-feed-broadcast/nodejs/infra/azure.bicep:1]: ok sure

Comment (Mohammed-MSFT) [reply samples/graph-activity-feed-broadcast/nodejs/infra/azure.bicep:1]: Removed bot related things.

File: samples/graph-activity-feed-broadcast/nodejs/.vscode/tasks.json [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+            // See https://aka.ms/teamsfx-tasks/local-tunnel for the detailed args definitions.
+            "label": "Start local tunnel",
+            "type": "teamsfx",
+            "command": "debug-start-local-tunnel",
+            "args": {
+                "type": "dev-tunnel",
+                "ports": [
+                    {
+                        "portNumber": 3978,
+                        "protocol": "https",
```
Comment (formulahendry) [inline samples/graph-activity-feed-broadcast/nodejs/.vscode/tasks.json:46]: why change to https?

**Review by formulahendry — APPROVED** [review]

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Please run build pipeline before merging this PR.

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**